From 4e572163e80abe8ce2545137eab8ec77dd0a189e Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 1 Oct 2019 22:01:34 +0200
Subject: [PATCH 03/23] COMP: Enable Slicer_INSTALL_DEVELOPMENT option

Currently the Slicer_INSTALL_DEVELOPMENT option is set to OFF. These
changes will enable Slicer_INSTALL_DEVELOPMENT as an option that can
be set ON/OFF.
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f4df0c64..9edb8a8bb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -350,9 +350,9 @@ mark_as_superbuild(Slicer_USE_SimpleITK_SHARED)
 # Install no development files by default, but allow the user to get
 # them installed by setting Slicer_INSTALL_DEVELOPMENT to true.
 #-----------------------------------------------------------------------------
-#option(Slicer_INSTALL_DEVELOPMENT "Install Slicer extension development files." OFF)
-#mark_as_advanced(Slicer_INSTALL_DEVELOPMENT)
-set(Slicer_INSTALL_DEVELOPMENT OFF)
+option(Slicer_INSTALL_DEVELOPMENT "Install Slicer extension development files." OFF)
+mark_as_advanced(Slicer_INSTALL_DEVELOPMENT)
+
 if(NOT Slicer_INSTALL_DEVELOPMENT)
   set(Slicer_INSTALL_NO_DEVELOPMENT 1)
 else()
-- 
2.24.1

